src/core/controller/index.ts

```diff
@@ -261,13 +261,6 @@ export class Controller {
 						}
 					}
 				})
-
```

Comment:
why are we removing this?

Reply:
no longer needed

Reply:
Can you explain a bit more as to why this is no longer needed?  What purpose did it provide that we no longer require?

Reply:
It's always been a bit redundant. But before when we did not opt the user in by default we had to trigger an opt in when the webview loaded. Now it just uses the user's saved preference

Reply:
OK, thanks for the info.